clients/client-python/gravitino/filesystem/gvfs.py

```diff
@@ -0,0 +1,574 @@
+        :return A file-like object from the filesystem
+        """
+        context: FilesetContext = self._get_fileset_context(path)
+        if context.storage_type == StorageType.HDFS:
+            return context.fs._open(
+                self._strip_storage_protocol(context.storage_type, context.actual_path),
+                mode,
+                block_size,
+                seekable,
+                **kwargs,
+            )
+        if context.storage_type == StorageType.FILE:
+            return context.fs._open(
+                self._strip_storage_protocol(context.storage_type, context.actual_path),
+                mode,
+                block_size,
+                **kwargs,
+            )
+        raise RuntimeError(f"Storage type:{context.storage_type} doesn't support now.")
+
+    def mkdir(self, path, create_parents=True, **kwargs):
+        """Make a directory.
+        :param path: Virtual fileset path
+        :param create_parents: Create parent directories if missing when set to True
+        :param kwargs: Extra args
+        """
+        context: FilesetContext = self._get_fileset_context(path)
+        context.fs.mkdir(
+            self._strip_storage_protocol(context.storage_type, context.actual_path),
+            create_parents,
+            **kwargs,
+        )
+
+    def makedirs(self, path, exist_ok=True):
+        """Make a directory.
+        :param path: Virtual fileset path
+        :param exist_ok: Continue if a directory already exists
+        """
+        context: FilesetContext = self._get_fileset_context(path)
+        context.fs.makedirs(
+            self._strip_storage_protocol(context.storage_type, context.actual_path),
+            exist_ok,
+        )
+
+    def created(self, path):
+        raise RuntimeError("Unsupported method now.")
+
+    def modified(self, path):
+        """Returns the modified time of the path file if it exists.
+        :param path: Virtual fileset path
+        :return Modified time(datetime.datetime)
+        """
+        context: FilesetContext = self._get_fileset_context(path)
+        return context.fs.modified(
+            self._strip_storage_protocol(context.storage_type, context.actual_path)
+        )
+
+    def cat_file(self, path, start=None, end=None, **kwargs):
+        """Get the content of a file.
+        :param path: Virtual fileset path
+        :param start:
+        :param end:
+            Bytes limits of the read. If negative, backwards from end,
+            like usual python slices. Either can be None for start or
+            end of file, respectively
+        :param kwargs: Extra args
+        :return File content
+        """
+        context: FilesetContext = self._get_fileset_context(path)
+        return context.fs.cat_file(
+            self._strip_storage_protocol(context.storage_type, context.actual_path),
+            start,
+            end,
+            **kwargs,
+        )
+
+    def get_file(self, rpath, lpath, **kwargs):
```

Comment:
Is this remote path a virtual path, can `lpath` be a virtual path/remote path or it can only be a local path, if so can we add some checks here?

Reply:
I think according to the design of fsspec, lpath can only be a local path.